dashboard/db/migrate/20151022174100_create_puzzle_ratings.rb

```diff
@@ -0,0 +1,15 @@
+class CreatePuzzleRatings < ActiveRecord::Migration
+  def change
+    create_table :puzzle_ratings do |t|
+      t.references :user, index: true
+      t.integer :script_id
```

Comment:
They're functionally identical, right? I just find it misleading that references by default create neither indexes nor foreign keys.